### PR TITLE
Add title on hover

### PR DESCRIPTION
**Pull request by KomalSrivastava**
# Issue Title: Add hover effect and title
Fixes issue #477 

## Type of change ☑️
I have added hover in each and the title of the tool , when the user point curser to the icon then it show the effect and the title or you can say the topic.
What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

 @singodiyashubham87 Review this and merge my issue.
## How Has This Been Tested? ⚙️

Describe how have you verified the changes made.

### CODE REVIEW

File: Draw-it-out
Comment (singodiyashubham87) [inline Draw-it-out:1]: You don't have to create a submodule.

**Review by singodiyashubham87 — CHANGES_REQUESTED** [review]
Do the changes.

**Comment by singodiyashubham87** [comment]
@KomalSrivastava You need to remove the submodules not add them.

**Comment by KomalSrivastava** [comment]
@singodiyashubham87 removed.

**Comment by KomalSrivastava** [comment]
@singodiyashubham87 Can you review it!!

**Comment by KomalSrivastava** [comment]
@singodiyashubham87 Review and merge it please !!

**Comment by KomalSrivastava** [comment]
@0xabdulkhalid @singodiyashubham87  ??

**Comment by singodiyashubham87** [comment]
@KomalSrivastava There is still a sub-module.
You should take care of all these things before creating a PR.
image

**Comment by KomalSrivastava** [comment]
@singodiyashubham87 I have removed .gitmodule may be haven't updated , my apology. now removed.

**Comment by KomalSrivastava** [comment]
@singodiyashubham87 i will create new PR . here error is showing

**Comment by singodiyashubham87** [comment]
@KomalSrivastava Go Ahead!

**Pull request closed without merging**